client/executor/common/src/error.rs

```diff
@@ -35,7 +33,7 @@ pub enum Error {
 	Wasmi(wasmi::Error),
 	/// Wasmtime action error
 	#[cfg(feature = "wasmtime")]
-	Wasmtime(ActionError),
+	Wasmtime(wasmtime_jit::ActionError),
```

Comment:
Why do we still have wasmi and wasmtime related error messages. I would like to have them removed.

Reply:
The engines still need to return error messages. How do you feel about returning the errors coming out of them as stringly-typed, i.e. `Other(String)`?

Reply:
Either this or boxing the error?